lib/checks/aria/has-global-aria-attribute-evaluate.js

```diff
@@ -0,0 +1,7 @@
+import lookupTable from '../../commons/aria/lookup-table';
```

Comment:
We recently released 4.0 in which we deprecated the lookup-table. You'll want to use the new `standards` object to get the [list of global attributes](https://github.com/dequelabs/axe-core/blob/develop/lib/commons/standards/get-global-aria-attrs.js)

Reply:
Resolved in PR